app/view/WarningView.js

```diff
@@ -104,7 +104,7 @@ SDL.warningView = Em.ContainerView
            * Check for webkit fillmode animation support Android < 4 version
            * doesnt support webkit animation fillmode
            */
-          checkForExternalConsent3AnimationSupport: function() {
+          checkForCCS3AnimationSupport: function() {

```

Comment:
@wolfylambova22 , i understand it is not in scope of your task, but can you change CCS to CSS. This is not your mistake (just misstake in code). Can you?